packages/stripe/pubspec.yaml

```diff
@@ -11,10 +11,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  stripe_android: ^0.0.1
-  stripe_ios: ^0.0.1
-  stripe_platform_interface: ^0.0.1
-
+  stripe_android: ^0.0.1+1
```

Comment:
shouldn't these become 1.0.1+1 as well just to try to keep things uniform?

Reply:
Yeah, I agree, let's do it for the next release. I already published this one